src/components/App.jsx

```diff
@@ -0,0 +1,29 @@
+import React, {Component, PropTypes} from "react"
+import {connect} from "react-redux"
+import {fetchCatsRequest} from "./../action_creators"
+import Cat from "./cat"
+
+class App extends Component {
+  componentWillMount () {
```

Comment:
This won't cause problems in your app but I'm more of a fan of putting any state updating actions in componentDidMount - you would normally have a loading component if you didn't have data yet, and then return the main component on load (through a condition checking props in the render function)